get_aqi.py

```diff
@@ -209,7 +279,7 @@ def publish_mqtt(mqtt_hostname, mqtt_port, mqtt_messages):
         # The tuples are of the form ("<topic>", "<payload>", qos, retain).
         # topic must be present and may not be empty
         msg_aqi = (topic + "/aqi", str(aqi), 0, False)
-        msg_level = (topic + "/level", str(aqi_level["level"]), 0, False)
+        msg_level = (topic + "/level", str(aqi_level), 0, False)
```

Comment:
`aqi_level` contains an entry `level` which contains a string representing the current AQI level.

For example : 

- `str(aqi_level)` -> `{'level': 'very low'}`
- `str(aqi_level["level"])` -> `very low`

As I'm looking to retrieve the AQI level string, I suggest this change :

```suggestion
        msg_level = (topic + "/level", str(aqi_level["level"]), 0, False)
```